AcadeMeData/tests/test_course.py

```diff
@@ -17,6 +15,11 @@ def generate_course(generate_degree, generate_professor, course_id=1, name="Hist
 class TestCourseModel:
     def test_get_course_by_name(self, generate_course, name="History of Countries"):
         course = Course.get_course_by_name(name)
-
         assert course.name == generate_course.name
         assert course.professor == generate_course.professor
+
+    def test_if_course_belongs(self, generate_course, generate_degree, generate_university):
+        assert generate_course.course_belongs(generate_university, generate_degree)
+
+    def test_if_course_not_belongs(self, generate_course, generate_university):
+        assert not generate_course.course_belongs(generate_university, None)
```

Comment:
The test is insufficient, you should check that with a degree that does not include the course, not with None.

Reply:
Added a degree instead of none @Omeramsc